admissions/models.py

```diff
@@ -0,0 +1,24 @@
+from django.db import models
+from common.util import get_semester_year_shorthand
+
+class Applicant(models.Model):
+    name = models.CharField(default="", null=False, max_length=100)
```

Comment:
Should probably split this as `first_name` and `last_name`. Can also remove the default becuase otherwise it will not force validation meaning that we can get applicants without a name